### ISSUE TITLE
Unable to upload media attachments

### ISSUE THREAD

**Issue by LiljebergXYZ** [neet/masto.js#97]
It appears that uploading media attachments is broken. I get the below error with both my own code and the original example form inside the project.

```
(node:19274) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null
    at FormData._getContentDisposition (/mnt/dev/bot/node_modules/form-data/lib/form_data.js:226:40)
    at FormData._multiPartHeader (/mnt/dev/bot/node_modules/form-data/lib/form_data.js:177:33)
    at FormData.append (/mnt/dev/bot/node_modules/form-data/lib/form_data.js:70:21)
    at createFormData (/mnt/dev/bot/node_modules/masto/dist/index.js:1:4494)
    at /mnt/dev/bot/node_modules/masto/dist/index.js:1:4447
    at Array.forEach (<anonymous>)
    at createFormData (/mnt/dev/bot/node_modules/masto/dist/index.js:1:4409)
    at /mnt/dev/bot/node_modules/masto/dist/index.js:1:4447
    at Array.forEach (<anonymous>)
    at createFormData (/mnt/dev/bot/node_modules/masto/dist/index.js:1:4409)
(node:19274) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:19274) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by neet** [comment]
Thanks for your report, This issue should've been fixed on the latest version [`2.5.3`](https://www.npmjs.com/package/masto/v/2.5.3). Please upgrade and let me know if it works properly?

**Comment by LiljebergXYZ** [comment]
Thanks for the fast update @neet, that appears to have done the trick!
